### DISCUSSION TITLE
Discussions now available here & some team administrivia.

### DISCUSSION THREAD

**Discussion by n2ygk** [django-oauth/django-oauth-toolkit#1479] [General]
# Discussions are now enabled
Hey everyone, Jazzband had a [team](https://github.com/orgs/jazzband/teams/django-oauth-toolkit) discussions feature which github apparently deprecated, so now our discussions can happen here, thanks to @jezdez enabling the feature. I've looked over the archive of those old team discussions and didn't see anything worth carrying over.

# Join the Jazzband and DOT team
Those of you who are regular contributors please do join the [Jazzband](https://jazzband.co/) and the [django-oauth-toolkit](https://jazzband.co/projects/django-oauth-toolkit) team.

# Team Lead(s) needed
Right now this project has two team leads, @n2ygk and @MattBlack85. Mattia's been otherwise busy for a while so we can definitely use one or two more team leads. Team leads are the only ones who can push new releases to pypi and it's important for a security-oriented project like DOT that the leads make sure malware doesn't sneak in to releases. See https://jazzband.co/about/releases#security for more about this.

**Comment by MattBlack85** [comment]
I think it is officially time to remove me from that list. I did nothing for the project in the last years and my personal interests brought me very very far from the python world

**Reply by n2ygk** [reply]
Sorry to see you go but I understand completely.

**Comment by n2ygk** [comment]
@dopry Are you interested in joining as a team lead? I see you are already a Jazzband member....

**Reply by dopry** [reply]
Absolutely

**Reply by n2ygk** [reply]
@dopry 
Please apply to be a project lead here: https://github.com/jazzband/help/issues/new?assignees=&labels=lead&projects=&template=project_lead.md&title=Project+Lead%3A+%3CPACKAGE+NAME%3E

**Reply by dopry** [reply]
done. are there any project specific team lead guidelines?

**Reply by n2ygk** [reply]
@dopry Just what's here: https://django-oauth-toolkit.readthedocs.io/en/latest/contributing.html#publishing-a-release, especially the part about comparing the distro files because checksums will not match.

When a release is ready and has been merged into `upstream`, add a tag in the form x.y.z and `push --tags upstream` which will trigger a Jazzband distro build. We'll both receive emails (along with jezdez) and then can follow up there.

Thanks for stepping up!

**Reply by dopry** [reply]
I can work with that.
